tests/embed/test_mpca.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import pytest
+from numpy import testing
+from scipy.io import loadmat
+from tensorly.tenalg import multi_mode_dot
+
+from kale.embed.mpca import MPCA
+
+gait = loadmat("tests/test_data/gait_gallery_data.mat")
```

Comment:
Would this be better as a fixture @drj11, @robadob, @ptheywood, @davidwilby? I'm a bit uncomfortable with not passing it as an argument to a function...

Reply:
Ideally you want a clean environment between tests, so one test doesn't infect the next. E.g. test a might break `gait`, making all subsequent tests fail.

If the cost of loading this stuff is huge (and would make running test suite impractical), could argue against that and just be very careful though.

A fixture would be a suitable way of automatically loading/deleting it per test, but I often get lazy and just copypaste my test setups between tests due to all the small per-test tweaks. Achieves the same thing, mostly a preference/standards thing.

Reply:
Thanks @robadob. I'll leave the decision to @sz144 in the context of my other comments.